Dockerfile

```diff
@@ -197,14 +197,19 @@ RUN . $NVM_DIR/nvm.sh \
 
 #------------------------------------------------------------------------
 # RUST - Build as a separate component
-FROM build AS rustbuild
+FROM ort-base-image AS rustbuild
```

Comment:
Unrelated change. Please split to separate commit.

Reply:
@MarcelBochtler How this is unrelated ?

Reply:
@MarcelBochtler  Not following you here as well ...